Source/FieldSolver/SpectralSolver/SpectralAlgorithms/PsatdAlgorithm.H

```diff
@@ -41,7 +41,8 @@ class PsatdAlgorithm : public SpectralBaseAlgorithm
             const amrex::Array<amrex::Real,3>& v_galilean,
             const amrex::Real dt,
             const bool update_with_rho,
-            const bool time_averaging);
+            const bool time_averaging,
+            const bool linear_in_J);
```

Comment:
Discussed in our meeting:
maybe better `psatd_linear_in_time`

Also: let us add doxygen strings that explain these flags/options :)
e.g. this one will use two J fields